src/main/java/com/google/devtools/build/lib/unix/jni/BUILD

```diff
@@ -0,0 +1,23 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+package(default_visibility = ["//src:__subpackages__"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//src/main/java/com/google/devtools/build/lib/unix:__pkg__"],
+)
+
+# Java interfaces to various native UNIX system services.
+#
+# All programs that depend on libraries in this package will have
+# JNI (native code) dependencies.
+java_library(
```

Comment:
Nothing is gained by splitting lib.unix.jni from lib.unix. Neither is useful without the other, and both are primarily concerned with "native functions for POSIX platforms", so they are not in any useful sense separate packages.

But in the interests of workflow efficiency, let's commit this change, which is mostly progress, and deal with the rest in a follow-up.

Ditto lib.windows.

Reply:
I've got a stack of changes pending for cleanups, I'll add this to the list.